src/Expensify.js

```diff
@@ -70,6 +73,7 @@ const defaultProps = {
         authToken: null,
         accountID: null,
     },
+    private_pushNotificationID: null,
```

Comment:
I think we should drop the `private_` prefix here

```suggestion
    pushNotificationID: null,
```

Reply:
Isn't it better to keep the same name as the onyx key and the same name as the NVP in the DB?
Why change it?

Reply:
I looked at some examples, e.g. `NVP_BLOCKED_FROM_CONCIERGE` and it seems we drop the `private_` prefix.

Reply:
Do you agree that's better? Because to me, keeping the same name everywhere sounds better than changing it slightly in some places.

Reply:
Yes, I agree it's better without the prefix

Reply:
ok updated